src/bench/streams_findbyte.cpp

```diff
@@ -14,20 +15,28 @@
 static void FindByte(benchmark::Bench& bench)
 {
     const auto testing_setup{MakeNoLogFileContext<const BasicTestingSetup>(ChainType::REGTEST)};
-    AutoFile file{fsbridge::fopen(testing_setup->m_path_root / "streams_tmp", "w+b")};
-    const size_t file_size = 200;
-    uint8_t data[file_size] = {0};
-    data[file_size - 1] = 1;
-    file << data;
+    constexpr size_t file_size{2 * MAX_BLOCK_SERIALIZED_SIZE};
+    constexpr size_t stride{32 << 10};
+    constexpr size_t found_index{file_size - 1};
+    constexpr std::byte target{1};
+
+    const auto path{testing_setup->m_path_root / "streams_tmp"};
+    AutoFile file{fsbridge::fopen(path, "w+b")};
+    std::vector data(file_size, std::byte{0});
+    data[found_index] = target;
+    file.write_buffer(data);
     file.seek(0, SEEK_SET);
     BufferedFile bf{file, /*nBufSize=*/file_size + 1, /*nRewindIn=*/file_size};
 
     bench.run([&] {
-        bf.SetPos(0);
-        bf.FindByte(std::byte(1));
+        for (size_t start{0}; start < file_size; start += stride) {
+            bf.SetPos(start);
+            bf.FindByte(target);
+            assert(bf.GetPos() == found_index);
+        }
     });
-
     assert(file.fclose() == 0);
+    fs::remove(path);
```

Comment:
the file isn't left around. This was fixed already in fa2fbaa4a29f80d3c7d5f0ad6b64035c3156dd12